src/app/Component/chatroom/chatroom.css

```diff
@@ -66,8 +66,8 @@
   
   .bounce2 {
   position: absolute;
-  bottom: 176px;
-  left: 15% ;
+  bottom: 54px;
+  left: 74px ;
```

Comment:
Why are you using Px.use %s wherever used px.px we will use upto 25 px.not more than 25